### PR TITLE
Input component

### PR DESCRIPTION
**Pull request by abhisheksahani-nova**
Completed input component with validation

### CODE REVIEW

File: Components/input/input.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
+.inp-container {
+  margin: 1rem;
+  position: relative;
+}
+
+.inp-label {
+  letter-spacing: 0.025rem;
+  color: #393d3d;
```
Comment (vrinda-mahajan) [inline Components/input/input.css:8]: Create variables for the colors.

**Review by vrinda-mahajan — CHANGES_REQUESTED** [review]

File: Components/input/input.css (hunk 1 above)
Comment (abhisheksahani-nova) [reply Components/input/input.css:8]: Yes vrinda I will once I start with documentation site , for now I have not decided my colors

**Review by vrinda-mahajan — APPROVED** [review]
Okay! The rest of the code looks good.

File: Components/input/input.css (hunk 1 above)
Comment (abhisheksahani-nova) [reply Components/input/input.css:8]: Thanks for the review

**Review by b0llu — APPROVED** [review]
Looks good to merge

**Pull request merged**